module/criteria/helper.js

```diff
@@ -57,7 +57,7 @@ module.exports = class criteriaHelper {
                     {_id : criteriaId},
                     {
                         rubric: rubric,
-                        criteriaType : "auto"
+                        criteriaType : existingCriteria.criteriaType ? existingCriteria.criteriaType : messageConstants.common.AUTO_RATING
```

Comment:
@deepakr123 - This will always be auto

Reply:
@deepakr123 - Add a condition to check only allowed levels are submitted in CSV.